Library/Formula/swiftrsrc.rb

```diff
@@ -0,0 +1,16 @@
+class Swiftrsrc < Formula
+  homepage "https://github.com/indragiek/swiftrsrc"
+  url "https://github.com/indragiek/swiftrsrc/archive/1.0.tar.gz"
+  sha1 "7fd484e0710c776a294896d93a2ee66c63069018"
+
+  depends_on "carthage" => :build
+
```

Comment:
` depends_on :macos => :mavericks ` as well, from the [release page notes](https://github.com/indragiek/swiftrsrc/releases). Potentially that's actually an ` xcode ` version requirement though, rather than a MacOS version requirement?